recipes/micropython/build.sh

```diff
@@ -0,0 +1,42 @@
+if [ `uname` != Darwin ]; then
+    sed -i 's/gcc/\$\(GCC\)/g' $SRC_DIR/py/mkenv.mk
+fi
+
+cd $SRC_DIR/lib
+sha=(
+    43a6e6bd3bbc03dc501e16b89fba0ef042ed3ea0
+    e9de7e35f2339598b16cbb375f9992643ed81209
+    35aaec4418ad78628a3b935885dd189d41ce779b
+)
+library=(
+    axtls
+    libffi
+    berkeley-db-1.xx
+)
+name=(
+    pfalcon
+    atgreen
+    pfalcon
+)
+for i in ${!library[@]}; do
+    rmdir ${library[$i]}
+    wget https://github.com/${name[$i]}/${library[$i]}/archive/${sha[$i]}.zip
+    unzip ${sha[$i]}.zip
+    mv ${library[$i]}-${sha[$i]} ${library[$i]}
+done
+
+cd $SRC_DIR/mpy-cross
+if [ `uname` == Darwin ]; then
+    CFLAGS_EXTRA="$CFLAGS" CPP="$CC -E" make
+else
+    CFLAGS_EXTRA="$CFLAGS" CPP="$GCC -E" make
+fi
```

Comment:
I think it can be just
```
CFLAGS_EXTRA="$CFLAGS" CPP="$CC -E" make
```

am I right? Because ``$CC`` on linux will point to gcc and on ``osx`` it will be ``clang``

Reply:
and is it possible to compile it using multiple threads?
```
CFLAGS_EXTRA="$CFLAGS" CPP="$CC -E" make -j${CPU_COUNT}
```